src/plugins/discover/public/application/components/default_discover_table/helper.tsx

```diff
@@ -55,11 +56,16 @@ export interface ColumnProps {
  * prepended, not moveable and removeable
  * @param timeFieldName
```

Comment:
i don't agree with having jsdoc in a typescript file, but since it's already here, lets add:
```suggestion
 * @param timeFieldName
 * @param osdFieldOverrides
```